charts/openshift-metering/templates/presto/presto-coordinator-config.yaml

```
@@ -59,6 +59,19 @@ data:
 {{- end }}
 
   jvm.config: |
+{{- if .Values.networking.useGlobalProxyNetworking }}
+{{- if .Values.presto.spec.config.tls.enabled }}
+    -Dhttp.proxyHost=networking.proxy.config.http_proxy.hostname
+    -Dhttp.proxyPort=networking.proxy.config.http_proxy.port
+    -Dhttp.proxyUser=networking.proxy.config.http_proxy.username
+    -Dhttp.proxyPassword=networking.proxy.config.http_proxy.password
+{{ else }}
+    -Dhttps.proxyHost=networking.proxy.config.https_proxy.hostname
+    -Dhttps.proxyPort=networking.proxy.config.https_proxy.port
+    -Dhttps.proxyUser=networking.proxy.config.https_proxy.username
+    -Dhttps.proxyPassword=networking.proxy.config.https_proxy.password
```

You need to add templating to inject the value of these properties
(this applies to this entire jvm configuration code block)
```suggestion
    -Dhttps.proxyPassword={{ .Values.networking.proxy.config.https_proxy.password }}
```

---

This still needs to be addressed.